src/gc/unix/gcenv.unix.cpp

```diff
@@ -517,6 +523,14 @@ void GCToOSInterface::GetMemoryStatus(uint32_t* memory_load, uint64_t* available
         *available_page_file = 0;
 }
 
+// Get size of an OS memory page
+void GCToOSInterface::GetPageSize()
+{
+    static pageSize = (sysconf( _SC_PAGE_SIZE ) != -1) ? sysconf( _SC_PAGE_SIZE ) : 4096;
```

Comment:
You can use the STL in this file - `std::call_once` is probably safer and more standard. Though +1 for it going into `GCToOSInterface::Initialize` instead.